fullrt/dht.go

```diff
@@ -42,8 +42,13 @@ import (
 	"github.com/libp2p/go-libp2p-xor/kademlia"
 	kadkey "github.com/libp2p/go-libp2p-xor/key"
 	"github.com/libp2p/go-libp2p-xor/trie"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 )
 
+var Tracer = otel.Tracer("")
```

Comment:
name?

Reply:
empty string uses the default tracer instance, which is fine because this isnt a top level application.

Reply:
Ah, got it.